lab_6_pipeline/pipeline.py

```diff
@@ -189,6 +304,14 @@ def _count_frequencies(self, article: Article) -> dict[str, int]:
         Returns:
             dict[str, int]: POS frequencies
         """
+        pos_freq = {}
+        for sentence in self._analyzer.from_conllu(article).sentences:
+            words = []
+            for word in sentence.words:
+                words.append(word.to_dict().get('upos'))
+            for word in set(words):
+                pos_freq[word] = pos_freq.get(word, 0) + words.count(word)
```

Comment:
nested loops is not good

can you rewrite to have only one for loop inside main loop?

Reply:
fixed